jobs/e2e_node/containerd/image-config-serial-resource-managers.yaml

```diff
@@ -3,14 +3,14 @@
 # `gcloud compute --project <to-project> images create <image-name> --source-disk=<image-name>`
 images:
   ubuntu:
-    image_family: pipeline-1-24
-    project: ubuntu-os-gke-cloud
+    image_family: ubuntu-2204-lts
```

Comment:
Out of curiosity, why stop use the GKE images ?

Reply:
Ok. I had a wrong assumption of `pipeline-1-24`. I thought since this family is ubuntu-based:

```bash
$gcloud compute images describe-from-family pipeline-1-24 --project ubuntu-os-gke-cloud --format='value(name,selfLink)'  
ubuntu-gke-2204-1-24-v20221128	https://www.googleapis.com/compute/v1/projects/ubuntu-os-gke-cloud/global/images/ubuntu-gke-2204-1-24-v20221128
```

the node(s)  will have cgroupsv2 enabled.